addon/helpers/class-names.js

```diff
@@ -2,5 +2,9 @@ import { helper } from '@ember/component/helper';
 import classnames from 'classnames';
 
 export default helper(function classNames(params, hash) {
-  return classnames(...params, hash);
+  const hashAsObject = {};
+  for (const key in hash) {
```

Comment:
Since `for ... in` enumerates over prototype methods too ought the solution be...

```javascript
const entries = Object.entries(hash);
const hash = Object.fromEntries(entries);

return classnames(...params, hash);
```